### DISCUSSION TITLE
Fix view stats

### DISCUSSION THREAD

**Discussion by QuartzWarrior** [yhype/yhype-feedback#29] [Bug] [answered]
## View stats have not been updated, while clones and stars are being updated.

views:
![views](https://github.com/yhype/yhype-feedback/assets/74436633/f5f5a410-7712-456a-bb94-223d6e4475f9)
clones:
![clones](https://github.com/yhype/yhype-feedback/assets/74436633/7e53eab4-0ddc-4979-b7d9-d9fb79827f1f)
stars:
![stars](https://github.com/yhype/yhype-feedback/assets/74436633/71640d9b-a0b4-46ca-ad25-bc09c5466f7e)

I assume this is just a me problem, as others seem to have it working in similar discussions. If this is the case, please let me know of a fix.

**Comment by antonkomarev** [comment]
@QuartzWarrior thank you for the report, gonna look at it closer next week.

**Reply by QuartzWarrior** [reply]
OK, thank you

**Reply by antonkomarev** [reply]
@QuartzWarrior could you show screenshot of the page:
https://github.com/QuartzWarrior/Tip.cc-Autocollect/graphs/traffic

I don't see any strange anomalies in code. GitHub just returns 0 views.

**Reply by antonkomarev** [reply, marked as answer]
@QuartzWarrior I suppose I found an issue, but don't know the reason yet. Let's look if simple daemons restart will help. Thank you for the report!

**Reply by QuartzWarrior** [reply]
Alright, I'll check. Here's the screenshot you asked for:

![Screenshot_20240428_115806_Brave.jpg](https://github.com/yhype/yhype-feedback/assets/74436633/def85a06-fb5b-443a-9434-e2b6e108146e)

**Reply by QuartzWarrior** [reply]
Looks to have been fixed on the 14th, odd.
![Screenshot_20240428_115646_Brave.jpg](https://github.com/yhype/yhype-feedback/assets/74436633/ffa14a6c-08cd-44d7-88a1-962e6f7d9593)
